model-optimizer/extensions/ops/gathernd.py

```diff
@@ -0,0 +1,103 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+from mo.front.common.partial_infer.utils import int64_array
+from mo.graph.graph import Node, Graph
+from mo.ops.op import Op
+
+
+class GatherND(Op):
+    op = 'GatherND'
+
+    def __init__(self, graph: Graph, attrs: dict):
+        mandatory_props = {
+            'kind': 'op',
+            'type': __class__.op,
```

Comment:
`__class__.` -> `self.`. The same comment for line 31 and 33.

Reply:
Fixed.